src/js/themes/colors.js

```diff
@@ -79,6 +79,10 @@ export const colors = {
     dark: dark.hpe.color.text.default,
     light: light.hpe.color.text.default,
   },
+  'text-xweak': {
```

Comment:
Should we add a comment to remind ourselves to deprecate this?

Reply:
Done.